app/services/show_transaction.service.js

```diff
@@ -39,8 +39,11 @@ class ShowTransactionService {
   }
 
   static _response (transaction) {
+    transaction.invoice = this._addTransactionTypeToInvoice(transaction.invoice)
```

Comment:
We're directly setting the value of `transaction.invoice` here which isn't consistent with how we set `signedChargeValue` later -- it'd be neater to use `Object.assign()` for both 🙂

Reply:
Hopefully, my changes cover this awesome suggestion!